HACKING.md

```diff
@@ -140,6 +140,24 @@ assuming they have enough permissions.
 oc create secret generic gcp-kola-tests-config --from-file=config=/path/to/kola-secret
 ```
 
+### [OPTIONAL] Creating Azure credentials configs
+
+If you want to run kola tests against Azure images you need to
+create a scret with an azureProfile.json and a file called and
+azure auth file. See the
+[kola docs](https://github.com/coreos/coreos-assembler/blob/main/docs/mantle/credentials.md#azure)
+for more information on those files.
+
+Once you have the azureAuth.json and azureProfile.json for connecting to Azure,
+create the secret in OpenShift:
```

Comment:
For prod, are these secrets in BitWarden?

Reply:
In that case, should we call it out?

Reply:
I think we either need to add it to every section about secrets or trust that people probably know where to grab them from. (I was just copying another section here and modifying it).

Reply:
Hmm, I thought we did consistently mention BitWarden after the recent migration. But indeed, it's only in some sections. I think it'd be good to either mention it consistently (or I guess, not mention it at all and have that knowledge live elsewhere).

Reply:
added a commit to add this information to missing sections